server/src/users.js

```diff
@@ -174,12 +174,47 @@ exports.tradeCode = function(code) {
     if (res.statusCode >= 200 && res.statusCode < 300) {
       return body;
     }
+    mozlog.warn('fxa-tradecode-failed', {status: res.statusCode});
     throw errors.badToken();
   });
 };
 
+exports.disconnectDevice = function(deviceId) {
```

Comment:
I'm pretty sure there's old functions in here that we should delete, from the old oauth flow.